tests/regression/echotest/exponent-operator-test-expected.echo

```diff
@@ -0,0 +1,12 @@
+ECHO: 25
+ECHO: 25
```

Comment:
It looks like pretty much all languages assign a higher priority to exponentiation than unary, so it behaves as -(5 ^ 2). I guess that becomes more obvious when looking at the actual [math representation](https://www.wolframalpha.com/input/?i=-5**2).

[Python](https://docs.python.org/3/reference/expressions.html#operator-precedence) gives it a higher priority than unary
```
$ python3
>>> -5**2
-25
>>> (-5)**2
25
```
[JavaScript](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Operator_Precedence#Table) (v10.15.2) assigns a lower priority but nodejs can't even parse it without parenthesis (hu?)
```
$ nodejs 
> 5**2
25
> (-5)**2
25
> -(5**2)
-25
> -5**2
-5**2
  ^^
SyntaxError: Unexpected token **
```

Reply:
it now executing unary after exponent, however I had to make an awkward exception for 5**-2 as the parser wasn't accepting it as valid syntax.
With my very limited experiance with bison I am unsure of a nicer way to resolve this, so have settled on this line:
https://github.com/seam345/openscad/blob/24b9d404b4309c18aaba4006d2f885d477606cdc/src/parser.y#L465

Reply:
Right, that does look like it might have other side effect. I'll have a look if there's a way without special cases.